modules/common/react/README.md

```diff
@@ -74,7 +74,13 @@ includes custom `data-*` attributes such as `data-test-id` to help facilitate au
 
 #### `containerElement: Element`
 
-> The element that contains the portal children when `portal` is true.
+> The containing element for Popper elements. The Popper uses {@link
+> https://reactjs.org/docs/portals.html Portals} to place the DOM elements of the Popper in a
+> different place in the DOM to prevent issues with overflowed containers. When the popper is
```

Comment:
Nit: "in a different place in the dom" > "As the last child of the specified container element"